content/pt-br/community/code-of-conduct.md

```diff
@@ -1,27 +1,30 @@
 ---
-title: Community
+title: Código de Conduta da Comunidade Kubernetes
 layout: basic
 cid: community
-css: /css/community.css
+community_styles_migrated: true
 ---
 
-<div class="community_main">
-<h1>Código de Conduta da Comunidade Kubernetes</h1>
-
+<div class="community-section" id="cncf-code-of-conduct-intro">
+<p>
 Kubernetes segue
-<a href="https://github.com/cncf/foundation/blob/master/code-of-conduct.md">CNCF Código de Conduta</a>.
+<a href="https://github.com/cncf/foundation/blob/main/code-of-conduct.md">CNCF Código de Conduta</a>.
 O texto do CNC CoC é reproduzido abaixo, a partir de
```

Comment:
Typo: CNC

Reply:
Fixed in [ceba277](https://github.com/kubernetes/website/pull/38407/commits/ceba2773cfd333a10d397444da856e89005babb7). Thanks.